addons/website_event_exhibitor/controllers/exhibitor.py

```diff
@@ -55,15 +52,17 @@ def _event_exhibitors_get_values(self, event, **searches):
             ])
 
         # search on countries
-        search_countries = self._get_search_countries(searches['countries'])
```

Comment:
Maybe we could keep its compatibility if people used those links, like
`` country_ids = request.httprequest.form.getlist('sponsor_country') or literal_eval(searches.get('countries', []))`` ?